ui/qml/harbour-amazfish.qml

```diff
@@ -147,7 +137,8 @@ ApplicationWindowPL
         signal unitNew(string name)
         onUnitNew: {
```

Comment:
Should we just remove this entire block then?

Reply:
You are right, looks much better when whole block is removed.

I added just comment because the checking onUnitNew seems to be reasonable, but leads to unexpected infinite loop.